nrf52-hal-common/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![no_std]
+
+extern crate cast;
+extern crate cortex_m;
+extern crate embedded_hal as hal;
+extern crate nb;
+extern crate void;
+
+#[cfg(feature = "52832")]
+pub extern crate nrf52 as target_device;
```

Comment:
Since this is used fairly often, I'm wondering if just `target` would be a better name. Just a though, I don't feel strongly about this.

Reply:
Yep, sounds good; done!